src/calendar/calendar-day.style.tsx

```diff
@@ -80,47 +68,60 @@ export const GrowDayCell = styled.div<GrowDayCellProps>`
     }}
 `;
 
-const DayCellBasic = styled.div`
-    position: relative;
+export const OverflowDisplay = styled.div<OverflowDisplayProps>`
```

Comment:
Right now from what i see, no matter what, there'll always be a circle on the start or end. Hence i think there is no need for a semi circle concept. But again I could be wrong. We can discuss